src/game/client/components/menus.cpp

```diff
@@ -1473,21 +1478,23 @@ void CMenus::UpdateVideoModeSettings()
 
 int CMenus::GetInitAmount() const
 {
-	return 10;
+	int NumMenuImages = 0;
+	Storage()->ListDirectory(IStorage::TYPE_ALL, "ui/menuimages", MenuImageCountingScan, &NumMenuImages);
```

Comment:
This feels way over the top. To search through the directories just to grab a non-important number.